utils/trivia.py

```diff
@@ -78,31 +78,62 @@ class Trivia:
     inProgress: bool
 
     def __init__(self):
+        
+        self.qAsked = QuestionAnswer("Question asked", "Answer asked")
+
+        # This line prepopulates the question list and is only here to facilitate
+        # ease of development. Had development continued far enough, this line would
+        # be replaced with code that would read in the questions and answers from an
+        # external source.
         self.qlist = QuestionList([QuestionAnswer("Question 1", "Answer 1")])
+        
         self.inProgress = False
+        
+    ## This function is responsible for starting the trivia game.
+    # Return non-zero value to indicate failure
+    # Return 0 to indicate success
+    def start_game(self) -> int:
```

Comment:
The method name changes will require test changes, but I will take care of that later today.